osi_lane.proto

```diff
@@ -376,21 +376,44 @@ message Lane
         //
         optional Subtype subtype = 12;
 
-        // Lane boundaries which mark the outer borders of intersections towards
-        // non-drivable space, for example sidewalks.
+        // Lane boundaries which mark the outer boundaries of intersections towards
+        // space which is not qualified for PKW driving, for example borders, shoulders or sidewalks.
         // 
-        // They must share a \c LaneBoundary::boundary_line (i.e. a boundary point) 
-        // which is linked as left_lane_boundary_id or right_lane_boundary_id from 
-        // a lane of TYPE_DRIVING. 
-        //
-        // note The intersection area is defined as the area spanned by those 
-        // boundaries and straight lines over the successor/antecessor lanes of \c #type =
-        // \c #DRIVING. TODO: pictures, rules for start/end points?
+        // Qualified for PKW driving are logical lanes of type normal, i.e. LogicalLane.Type = TYPE_NORMAL.
+        // 
+        // Outer boundaries of intersections must share the position value of one 
+        // LogicalLaneBoundary.LogicalBoundaryPoint with the LogicalLaneBoundary.LogicalBoundaryPoint
+        // of another LogicalLaneBoundary which is linked as left_lane_boundary_id or right_lane_boundary_id from a 
+        // successor or predecessor logical lane with LogicalLane.Type = TYPE_NORMAL.
+        //
```

Comment:
Just mentioning.. there might be undefined areas between lane and border that are not part of the polygon if using the boundaries of the drivng lanes of the intersection.

Reply:
That's true. Thought about specifying those but got away from it (see https://github.com/OpenSimulationInterface/open-simulation-interface/pull/602/commits/ae06ee1d9a74744e41c09d119f8d47bb2018f12d) because at least when converting from OpenDRIVE (and that was the starting point for the logical lanes) the gaps are meaningless because they don't tell you if those areas are drivable or not. I would wait for OpenDRIVE to solve this problem prior specifying it here.